### PR TITLE
Userlocation 017

### PR DESCRIPTION
**Pull request by khc196**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -9,9 +9,9 @@
 
 """
 from __future__ import print_function
-import json
-import os
-import uuid, redis, yelpCrawler, Decorator_for_HTTP
+import json, os, uuid, redis, pymysql
+# import custom modules
```
Comment (dexterastin) [inline app.py:13]: remove this line

File: app.py [hunk 2]
```diff
@@ -43,124 +45,115 @@ def webhook():
...
+    if "cosy" in parameter_atmosphere :
+         atmosphere[2] = "cosy"
+    if "romantic" in parameter_atmosphere :
+         atmosphere[3] = "romantic"
+    if "classy" in parameter_atmosphere :
+         atmosphere[4] = "classy"
+    if "trendy" in parameter_atmosphere :
+         atmosphere[5] = "trendy"
+    if "hipster" in parameter_atmosphere :
+         atmosphere[6] = "hipster"
```
Comment (dexterastin) [inline app.py]: remove extra line plz

**Review by dexterastin — COMMENTED** [review]
Cool!

File: app.py (hunk 1 above)
Comment (khc196) [reply app.py:13]: Oh.. It's just explanation.

File: app.py (hunk 2 above)
Comment (khc196) [reply app.py]: Okay.

**Review by dexterastin — APPROVED** [review]
Cool!!

**Pull request merged**